add more details/notes for the mortgage perforamcne tests (#229)
NVIDIA/spark-rapids-examples: examples/XGBoost-Examples/README.md

![mortgage-speedup](/docs/img/guides/mortgage-perf.png)

Note that the test result is based on 21 years [Fannie Mea Single-Family Loan Performance Data](https://capitalmarkets.fanniemae.com/credit-risk-transfer/single-family-credit-risk-transfer/fannie-mae-single-family-loan-performance-data) 

> with a 4 A100 GPUs and 512 CPU vcores cluster, the performance is influenced by many conditions, 

Reviewer: ```suggestion
with a 4 A100 GPU and 512 CPU vcores cluster, the performance is affected by many aspects, 
```

> including the dataset size or the GPU card. If the application ran for too long or even failed, 

Reviewer: ```suggestion
including data size and type of GPU. 
```

Reviewer: I would suggest we delete the last sentence. And the previous words should be good enough.

> please tweak the configs according to your cluster.

Reviewer: ```suggestion
```
